CHANGELOG.md

```diff
@@ -1,167 +1,57 @@
-# [5.1.0](https://github.com/fourkitchens/sous-drupal-project/compare/5.0.0...5.1.0) (2023-04-27)
-
-
-### Features
-
-* add npm scripts to lando so ci and lando can run both seperately ([#102](https://github.com/fourkitchens/sous-drupal-project/issues/102)) ([412d607](https://github.com/fourkitchens/sous-drupal-project/commit/412d6073ea7b010c814748fc9cd6eadaa24db8d1))
-
-# [5.0.0](https://github.com/fourkitchens/sous-drupal-project/compare/4.4.0...5.0.0) (2023-04-21)
+# Changelog
 
+All notable changes specific to pantheon-upstreams/drupal-composer-managed are noted here.
```

Comment:
Why is the sous changelog overwritten with drupal-composer-managed changelog?

Reply:
@laura-johnson Could it be a change from when @cbfannin merge the Pantheon repository with ours?

Reply:
Could be! I think we want the sous version though...

Reply:
@laura-johnson @randallquesadaa Original changelog restored. Does that make this ready to merge? i.e. someone has functionally tested this?